packages/@azure/servicebus/data-plane/lib/core/messageReceiver.ts

```diff
@@ -252,8 +245,6 @@ export class MessageReceiver extends LinkEntity {
     if (!options) options = {};
     this.receiverType = receiverType;
     this.receiveMode = options.receiveMode || ReceiveMode.peekLock;
-    this.maxConcurrentCalls =
-      options.maxConcurrentCalls != undefined ? options.maxConcurrentCalls : 1;
```

Comment:
Why did we remove this?
The `maxConcurrentCalls` that is being passed in the `options` when StreamingReceiver is intantiated in receiver.ts file will never reach rhea without this line

Reply:
Oops yes, updated